contrib/OtlpHttp/SpanConverter.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\OtlpHttp;
+
+use Opentelemetry\Proto;
+use Opentelemetry\Proto\Common\V1\AnyValue;
+use Opentelemetry\Proto\Common\V1\ArrayValue;
+use Opentelemetry\Proto\Common\V1\InstrumentationLibrary;
+use Opentelemetry\Proto\Common\V1\KeyValue;
+use Opentelemetry\Proto\Trace\V1\InstrumentationLibrarySpans;
+use Opentelemetry\Proto\Trace\V1\ResourceSpans;
+use Opentelemetry\Proto\Trace\V1\Span as CollectorSpan;
+use Opentelemetry\Proto\Trace\V1\Span\Event;
+use Opentelemetry\Proto\Trace\V1\Span\SpanKind;
+use Opentelemetry\Proto\Trace\V1\Status;
+use Opentelemetry\Proto\Trace\V1\Status\StatusCode;
+use OpenTelemetry\Sdk\Trace\Span;
+use OpenTelemetry\Sdk\Trace\SpanStatus;
+
+class SpanConverter
+{
+    public function as_otlp_key_value($key, $value): KeyValue
+    {
+        return new KeyValue([
+            'key' => $key,
+            'value' => $this->as_otlp_any_value($value),
+        ]);
+    }
+
+    public function as_otlp_any_value($value): AnyValue
+    {
+        $result = new AnyValue();
+
+        switch (true) {
+            case is_array($value):
+
+                $values = [];
+                foreach ($value as $element) {
+                    $this->as_otlp_any_value($element);
```

Comment:
Is this recursive on purpose?

Reply:
To be completely honest, I copied it from here: https://github.com/open-telemetry/opentelemetry-ruby/blob/424cb23cb17f541ec4fa1be19b303ce04c773d91/exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb#L362

I believe it's designed it walk down nested arrays, and convert those into OTLP.